intro/scipy/index.rst

```diff
@@ -610,143 +610,105 @@ Statistics and random numbers: :mod:`scipy.stats`
-
-    >>> np.mean(samples)     # doctest: +ELLIPSIS
-    -0.0452567074...
-
-The median another estimator of the center. It is the value with half of
-the observations below, and half above::
-
-    >>> np.median(samples)     # doctest: +ELLIPSIS
-    -0.0580280347...
-
-.. tip::
-
-   Unlike the mean, the median is not sensitive to the tails of the
-   distribution. It is `"robust"
-   <https://en.wikipedia.org/wiki/Robust_statistics>`_.
-
-.. topic:: Exercise: Compare mean and median on samples of a Gamma distribution
-   :class: green
-
-    Which one seems to be the best estimator of the center for the Gamma
-    distribution?
-
-
-The median is also the percentile 50, because 50% of the observation are
-below it::
-
-    >>> sp.stats.scoreatpercentile(samples, 50)     # doctest: +ELLIPSIS
-    -0.0580280347...
-
-Similarly, we can calculate the percentile 90::
-
-    >>> sp.stats.scoreatpercentile(samples, 90)     # doctest: +ELLIPSIS
-    1.2315935511...
-
-.. tip::
-
-    The percentile is an estimator of the CDF: cumulative distribution
-    function.
-
-Statistical tests
-.................
-
-.. image:: auto_examples/images/sphx_glr_plot_t_test_001.png
-    :target: auto_examples/plot_t_test.html
-    :scale: 60
-    :align: right
-
-A statistical test is a decision indicator. For instance, if we have two
-sets of observations, that we assume are generated from Gaussian
-processes, we can use a
-`T-test <https://en.wikipedia.org/wiki/Student%27s_t-test>`__ to decide
-whether the means of two sets of observations are significantly different::
-
-    >>> a = np.random.normal(0, 1, size=100)
-    >>> b = np.random.normal(1, 1, size=10)
-    >>> sp.stats.ttest_ind(a, b)   # doctest: +SKIP
-    (array(-3.177574054...), 0.0019370639...)
-
-.. tip:: The resulting output is composed of:
-
-    * The T statistic value: it is a number the sign of which is
-      proportional to the difference between the two random processes and
-      the magnitude is related to the significance of this difference.
-
-    * the *p value*: the probability of both processes being identical. If it
-      is close to 1, the two process are almost certainly identical.
-      The closer it is to zero, the more likely it is that the processes
-      have different means.
-
-.. seealso::
-
-   The chapter on :ref:`statistics <statistics>` introduces much more
-   elaborate tools for statistical testing and statistical data
-   loading and visualization outside of scipy.
+   parameter of 1. *Hint: the shape parameter is passed as the first
+   argument when freezing the distribution*. Plot the histogram of the
+   sample, and overlay the distribution's PDF. Estimate the shape parameter
+   from the sample using the ``fit`` method.
+
+   Extra: the distributions have many useful methods. Explore them
+   using tab completion. Plot the cumulative density function of the
+   distribution, and compute the variance.
+
+Sample Statistics and Hypothesis Tests
```

Comment:
Decided to simplify this. The advanced section can elaborate. Here, I think it's important to be concise and correct, and these are challenging to achieve simultaneously if we go much deeper.